packages/file-store/test.ts

```diff
@@ -7,9 +7,10 @@ import path from 'node:path'
 import sinon from 'sinon'
 
 import {FileStore} from './'
-import {Upload, MemoryConfigstore} from '@tus/server'
+import {Upload} from '@tus/server'
```

Comment:
I think we needed the `@tus/server` dev dependency because of this usage in tests.

Reply:
I belive `@tus/server` should be listed as a dependency because `FileStore` extends `DataStore` and uses other types from `@tus/server`. 
```
import {DataStore, Upload, ERRORS} from '@tus/server'
```

Sure, `@tus/server` is also used in tests but since it is declared as a dependency I don't think we need to repeat that in `devDependencies`.

Reply:
I think it makes more sense to have peer dependency for how people consume this package and a dev dependency for our tests. What do you think?